services/ui-src/src/routes/Intro/Intro.tsx

```diff
@@ -1,29 +1,138 @@
 // components
-import { Box, Heading, Text } from "@chakra-ui/react";
-import { ReportPage } from "components";
+import { Box, Button, Image, Heading, Text, Flex } from "@chakra-ui/react";
+import { BasicPage, IntroSection, SpreadsheetWidget } from "components";
+import { useNavigate } from "react-router-dom";
+// utils
+import { makeMediaQueryClasses } from "utils";
 import verbiage from "verbiage/pages/mcpar/mcpar-intro";
+// assets
+import nextIcon from "assets/icons/icon_next.png";
 
 export const Intro = () => {
-  const { intro } = verbiage;
+  const { intro, body, pageLink } = verbiage;
+  const mqClasses = makeMediaQueryClasses();
+  const navigate = useNavigate();
+
+  const [section1, section2, section3] = body.sections;
+
   return (
-    <ReportPage data-testid="intro-view">
+    <BasicPage sx={sx.layout} data-testid="intro-view">
       <Box sx={sx.leadTextBox}>
         <Heading as="h1" sx={sx.headerText}>
           {intro.header}
         </Heading>
-        <Text>{intro.body}</Text>
       </Box>
-    </ReportPage>
+      <div>
+        <IntroSection index={1} content={section1} key={`section-1`}>
+          <Flex sx={sx.widgetsContainer} className={mqClasses}>
+            <Box sx={sx.imagelessWidgetContainer}>
+              <Text sx={sx.imagelessWidgetTitle}>{section1.widget?.title}</Text>
+              <Box>
+                {section1.widget?.descriptionList.map((description, index) => (
+                  <Text key={index}>{description}</Text>
+                ))}
```

Comment:
Typically want to avoid using `index` as key, and I think react will usually yell at you for it. I'd suggest just using the `description` as the key.

Reply:
using description would be a really long key.. maybe `section-1-desc-${index}`

Reply:
Yeah I mean even `index` is fine as a key in this instance because the children are not dynamically changing their order or having items added or deleted, which is really why you don't use index, because on re-renders it can cause bugs. In this case we can just use `index` though. 

In your suggestion the issue above could still happen because the only unique part is the index. As for description being real long, that's true, but also 🤷 it's in the DOM and computers don't care.

So for now, just key={index} is fine. 👍